### PR TITLE
Added ES6 feature support to NodeJS Unirest codegen

### PR DESCRIPTION
**Pull request by sastava007**
**What this PR does**
It allows the user to generate the code snippet for NodeJS Unirest in the ES6 format.
This is in response to issue #115  requested by one of the users.

**Changes you made**

- Registered a new parameter ES6_enabled in **`options()`** which allows the user to toggle between whether codegen generates snippet with ES6 features or not.

- By default ES6_enabled is set to `false`

- Also added the newly created option parameter in **`structure.test.js`** file to register it as a valid option id.

- Added unit tests to verify the generated snippets in **snippet.test.js**

- Added a new mocha test suite to verify that the generated snippet is runnable in **newman.test.js.**

- Updated the **readme.md** file with new option parameter.

**Screenshot/ CodeSnippet**
Not required

**Test Configuration**:

- Added a unit test to check the snippet generated

- Added a new block of mocha test inside the main suite to verify that the snippet produced is runnable.

**Follow up**
Not required

*Thanks for contributing!* ❤️

### CODE REVIEW

**Comment by sastava007** [comment]
@umeshp7 @shreys7 
Please review!

File: codegens/nodejs-unirest/test/unit/snippet.test.js [hunk 1]
```diff
@@ -48,6 +48,24 @@ describe('nodejs unirest convert function', function () {
...
+      convert(request, options, function (error, snippet) {
+        if (error) {
+          expect.fail(null, null, error);
+          return;
+        }
+        expect(snippet).to.be.a('string');
+        snippetArray = snippet.split('\n');
+        expect(snippetArray[0]).to.equal('const unirest = require(\'unirest\');');
+        expect(snippetArray).to.include(`const req = unirest('${mainCollection.item[0].request.method}', ` +
+        `'${mainCollection.item[0].request.url.raw}')`);
```
Comment (shreys7) [inline codegens/nodejs-unirest/test/unit/snippet.test.js]: Can you also add an expect for `.end` arrow function?

Comment (sastava007) [reply codegens/nodejs-unirest/test/unit/snippet.test.js]: @shreys7 Actually, initially I tried for that but don't know somehow I was getting error maybe because of some extra space. 
Let me try again.

Comment (sastava007) [reply codegens/nodejs-unirest/test/unit/snippet.test.js]: @shreys7 I have updated, please have a look.
